code/modules/clothing/under/jobs/security.dm

```diff
@@ -150,11 +150,11 @@
 	contained_sprite = TRUE
 
 /*
- * Head of Security
+ * Chief of Security
  */
 /obj/item/clothing/under/rank/head_of_security
```

Comment:
Idk if it matters. But these typepaths will now be misleading. We changed all the HoP ones to XO for the NBT I think?